scripts/aperture_dev_cluster.sh

```diff
@@ -0,0 +1,229 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+# Values selected due to minikube using them by default
+# If this is changed, update entries in GCP
+readonly cluster_subnet_prefix=192.168.49
+readonly cluster_subnet_mask=24
+readonly cluster_docker_network="ninja-cluster"
+readonly cluster_subnet="${cluster_subnet_prefix}.0/${cluster_subnet_mask}"
+readonly cluster_gateway="${cluster_subnet_prefix}.1"
```

Comment:
Since we don't have UI here (And don't configure contour at all), I'd honestly drop all of the docker-network related steps. It'll make it significantly smaller and simpler.

Reply:
We also have the internal docker registry which is been created and connected to the docker network . So if we are removing the docker-network steps . We will also need to remove the registry code right ? 
Which I think we will need for the 3 nodes setup

Reply:
We can keep the registry - it just needs to be hooked into the kind network then. Docs: https://kind.sigs.k8s.io/docs/user/local-registry/#create-a-cluster-and-registry
Simply replace `cluster_docker_network` value with `kind` and leave the registry code as-is, it should work just fine.

Reply:
Yeah, I made the changes and was testing But I saw that @seunghyupoh3517 , did a [PR](https://github.com/fluxninja/aperture/pull/259) where is mention how can we create 3 nodes cluster using `kind-config.yaml` file .  
So is this PR still relevant as we can create the 3 node cluster using this command
```
kind create cluster --name aperture-playground --config kind-config.yaml
```

Reply:
Yes, I needed to three nodes cluster for the rate limiter testing so I created the simple configuration file while discussing with @tanveergill. With the `kind-config.yaml`, we now have a control plane and two worker nodes kind cluster.

Reply:
So is this PR still relevant . Do we need the script to start the three node cluster or just kind-cluster.yaml file works ?

Reply:
kind-cluster.yaml file works to start the three node cluster but I haven't taken a look into rest of the codes in the scripts as well as the script in the cloud repo so I cannot tell if it's still needed.

Reply:
@fluxninja/devops know more about the requirements of 
cluster for Tilt environment. Please make a call on whether we need a script vs bare kind config will do